mesh_client/__init__.py

```diff
@@ -472,7 +553,29 @@ def __exit__(self, type_, value, tb):
         self.close()
 
 
-class Message(object):
+@dataclass
+class _MessageAttrs:
+    """
+    this is for type hinting on the commonly used message attrs below ( which are dynamically generated )
+    """
+
+    message_id: str
+    message_type: str
+    recipient: str
+    sender: Optional[str] = None
+
+    workflow_id: Optional[str] = None
+    filename: Optional[str] = None
+    local_id: Optional[str] = None
+    partner_id: Optional[str] = None
+    chunk_range: Optional[str] = None
+
+    subject: Optional[str] = None
+    encrypted: Optional[Union[str, bool]] = None
+    compressed: Optional[Union[str, bool]] = None
+
+
+class _BaseMessage:
```

Comment:
again the doc string looks like it needs updating here. missing checksum i think

Reply:
thanks